### DISCUSSION TITLE
Ethos 1.5 x12s

### DISCUSSION THREAD

**Discussion by spvwrx** [FrSkyRC/ETHOS-Feedback-Community#3171] [General]
Will there be an Ethos 1.5 version for Horus X12S?

**Comment by KevinFairgrieve** [comment]
I really do hope so, but I expect that as the Horus technology is now getting on, the physical would be the barrier and not ETHOS.

I plan to convert my second X12S to EDGE with touch screen like my other one if ETHOS is not supported.

My BETA X12S unit will be forever on OpenTX (hated FROS when it was being tested) as it is a bit of history.

As the 1.4 nightlies show the X12S maybe @bsongis-frsky could do a final version that would incorporate all the possible changes.

**Comment by Lefish63** [comment]
Being an X12s devotee due to ergonomics for my shovel size hands I have done the touch mod on my primary and back up radios, my third is still running Ethos, although the touch mod is installed.
Having had rotary encoder issues and an intermittent sticking enter button problem, which was potentially disastrious on Otx, touch was the way to go to reduce wear on the these items.
I would love to use Ethos, but no touch support makes it a non starter after experiencing my rotary encoder issues.
I do understand its a non Frsky upgrade so will probably never be supported. 
I use fairly simple model programming and pre 1.5.0 releases more than cover what I use it for but it would still be nice to have 1.5.0. supported for the more complex users although it's looking like lot of extra work ?

However as I have been exploring Edge the touch is excellent and it's now going to take a lot for me to move away from it now. 

Dave

**Reply by AeroSeb** [reply]
I have just bought an X10S ISRM and  install Ethos 1.4.15 on it.
I really need to have Ethos 1.5 on my Horus.
So I hope Frsky will make a special version for the X10/X12 radio even if these radio are old but X10 Express area always in sale...

**Comment by bsongis-frsky** [comment]
Available now

**Comment by spvwrx** [comment]
Thank you..Best regards,Simon VentevogelOn 17 Jan 2024, at 9:01 am, Bertrand Songis @ FrSky ***@***.***> wrote:﻿
Available now
